Update doc and typings for res.json() (#56)
fastify/light-my-request: README.md

    - `body` - alias for payload.
    - `rawPayload` - the raw payload as a Buffer.
    - `trailers` - an object containing the response trailers.

>     - `json` - a function that parses string to json.

Reviewer: I would improve the description:
`json()` - parse the application/json response payload and returns an object. Throws if the content type is different

>     - `json` - a function parses the JSON format payload.

Reviewer 2: ```suggestion
    - `json` - a function that parses string to json.
```
